src/scripts/Sitemap.js

```diff
@@ -1,22 +1,29 @@
-import * as Papa from 'papaparse';
 import DatePatternSupport from './DateUtils/DatePatternSupport';
 import SelectorList from './SelectorList';
 import Model from './Model';
+import SitemapSpecMigration from './SitemapSpecMigration';
+import * as browser from 'webextension-polyfill';
 
 export default class Sitemap {
-	constructor(id, startUrls, model, selectors) {
+	constructor(id, startUrls, model, selectors, sitemapSpecificationVersion = null) {
+		this.rootSelector = { id: '_root', uuid: '0' };
+		this.sitemapSpecificationVersion =
+			sitemapSpecificationVersion !== null ? sitemapSpecificationVersion : 0;
```

Comment:
I believe there is no need in `sitemapSpecificationVersion` argument since you should only work with latest version sitemaps.